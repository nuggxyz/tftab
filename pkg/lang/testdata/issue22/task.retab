gen taskfile {
	path = "./tmp.yaml"
	data = {
		cow = GO_MODULE_PACKAGE
		wut = SUP
	}
}
