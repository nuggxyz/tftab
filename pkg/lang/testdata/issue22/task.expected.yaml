# code generated by retab v0.0.0-unknown. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

cow: github.com/test/123
wut: NOPE
